.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: scala
```

Comment:
Is this the negative case u added to test CI?

Reply:
Nope. This is a working one ! you could see the build report here https://github.com/airavata-courses/CAPtivate/pull/19

Reply:
The green check mark next to the commit id indicates CI passed for the commit . It failed for the first commit since the api-gateway service wasn't there in the master then

Reply:
Ok got it, yes i have seen the build report. Thanks